src/main/java/cloud/fogbow/ras/core/models/orders/AttachmentOrder.java

```diff
@@ -82,14 +82,4 @@ public ResourceType getType() {
     public String getSpec() {
```

Comment:
Please, add a TODO here and a task to solve this Spec stuff. This will be needed in the accounting service.

Reply:
Added.